as-vending/Makefile

```diff
@@ -34,7 +34,7 @@ stop:
 	docker rm -f $(MICROSERVICE):latest
 
 test: lint
-	go test -test.v -cover ./...
+	go test -test.v -cover -count=1 -race ./... 
```

Comment:
Why `-count=1` ?

Reply:
we were seeing some caching issues so @jim-wang-intel suggested using -count=1 to clear cache